doc/nrf/known_issues.rst

```diff
@@ -916,6 +916,16 @@ NCSDK-12020: Current consumption for Gaming Mouse increased by 1400mA
 
   **Workaround:** Change ``pwm_pin_set_cycles`` to ``pwm_pin_set_usec`` in function :c:func:`led_pwm_set_brightness` in Zephyr's driver :file:`led_pwm.c` file.
 
+nRF Machine Learning
+====================
+
+.. rst-class:: v1-9-0
+
+NCSDK-13923: Device may crash during Bluetooth bonding
+  The device programmed with the nRF Machine Learning application may crash during Bluetooth bonding  because of insufficient Bluetooth RX thread stack size.
```

Comment:
```suggestion
  The device programmed with the nRF Machine Learning application may crash during Bluetooth bonding because of insufficient Bluetooth RX thread stack size.
```
Just a nit...

Reply:
Thanks